salt/states/boto_cognitoidentity.py

```diff
@@ -270,12 +261,12 @@ def pool_present(
             updated_identity_pool = r.get("identity_pool")
             IdentityPoolId = updated_identity_pool.get("IdentityPoolId")
             ret["comment"] = (
-                "A new identity pool with name {0}, id {1} "
+                "A new identity pool with name {}, id {} "
                 "is created.".format(IdentityPoolName, IdentityPoolId)
             )
         else:
             ret["result"] = False
-            ret["comment"] = "Failed to add a new identity pool: " "{0}".format(
+            ret["comment"] = "Failed to add a new identity pool: " "{}".format(
```

Comment:
```suggestion
            ret["comment"] = "Failed to add a new identity pool: {}".format(
```
and any other occurrences like this one on the file